teaspoons-cli/teaspoons/config.py

```diff
@@ -0,0 +1,22 @@
+# config.py
+
+from pathlib import Path
+from oauth2_cli_auth import OAuth2ClientInfo
+from dotenv import dotenv_values
+
+
+class CliConfig:
+    """A class to hold configuration information for the CLI"""
+
+    def __init__(self):
+        self.config = dotenv_values("../.env")
+
+        self.client_info = OAuth2ClientInfo.from_oidc_endpoint(
+            self.config["OAUTH_OPENID_CONFIGURATION_URI"],
+            client_id=self.config["OAUTH_CLIENT_ID"],
+            scopes=[f"openid+email+profile+{self.config['OAUTH_CLIENT_ID']}"]
```

Comment:
Does the client ID truly need to be passed as a scope, or does it work with just `openid+email+profile`?

Reply:
Oh okay, good to know that's still a requirement. It always felt weird to me.

I think the `openid email profile` scopes are configured in the B2C claims provider ([google](https://github.com/broadinstitute/terraform-ap-deployments/blob/master/azure/b2c/policies/TrustFrameworkExtensions.xml.tftpl#L127) and [azure](https://github.com/broadinstitute/terraform-ap-deployments/blob/master/azure/b2c/policies/TrustFrameworkExtensions.xml.tftpl#L228)). So maybe that's why just specifying the client ID here works. I think leaving it as-is is best though. 👍